### PR TITLE
Restyle

### PR DESCRIPTION
**Pull request by LegitTalon**
Lil bit o refact lil bit o restyle

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/LegitTalon/legittalon.github.io/pull/8" target="_blank">ReviewNinja

File: index.html [hunk 1]
```diff
@@ -2,11 +2,11 @@
 layout: default
 ---
 
-<div class="home">
+<div class="Page">
```
Comment (LegitTalon) [inline index.html]: This is already defined in `_layout/default.html` !fix

File: _includes/head.html [hunk 2]
```diff
@@ -1,13 +1,39 @@
...
+  />
 
-  <link rel="stylesheet" href="{{ "/css/main.css" | prepend: site.baseurl }}">
-  <link rel="stylesheet" href="{{ "/bower_components/font-awesome/css/font-awesome.min.css" | prepend: site.baseurl }}">
-  <link rel="canonical" href="{{ page.url | replace:'index.html','' | prepend: site.baseurl | prepend: site.url }}">
-  <link rel="alternate" type="application/rss+xml" title="{{ site.title }}" href="{{ "/feed.xml" | prepend: site.baseurl | prepend: site.url }}" />
+  <link rel="stylesheet" href="{{ "/css/application.css" | prepend: site.baseurl }}"/>
+  <link
+    rel="stylesheet"
+    href="{{"/bower_components/font-awesome/css/font-awesome.min.css" | prepend: site.baseurl}}"
```
Comment (LegitTalon) [inline _includes/head.html]: Remove this if font-awesome goes unused, which it might. !fix

File: _includes/footer.html [hunk 3]
```diff
@@ -1,54 +1,2 @@
...
-      </div>
-
-      <div class="footer-col  footer-col-3">
-        <p class="text">{{ site.description }}</p>
-      </div>
-    </div>
-
-  </div>
-
+<footer class="Footer">
```
Comment (LegitTalon) [inline _includes/footer.html]: Figure out what to put in the footer if anything. !resolve

File: _includes/head.html (hunk 2 above)
Comment (LegitTalon) [inline _includes/head.html]: !resolved

Comment (LegitTalon) [inline _includes/head.html]: !resolve

File: index.html (hunk 1 above)
Comment (LegitTalon) [inline index.html]: !fixed

File: _includes/footer.html (hunk 3 above)
Comment (LegitTalon) [inline _includes/footer.html]: !resolved

File: _includes/head.html (hunk 2 above)
Comment (LegitTalon) [inline _includes/head.html]: !resolved

Comment (LegitTalon) [inline _includes/head.html]: !resolved

Comment (LegitTalon) [inline _includes/head.html]: !resolved

**Pull request merged**